src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/FormTheme/category.html.twig

```diff
@@ -0,0 +1,29 @@
+{% extends '@PrestaShop/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig' %}
+
+{% block category_cover_image_widget %}
+  {% include '@PrestaShop/Admin/Sell/Catalog/Categories/Blocks/cover_image.html.twig' %}
```

Comment:
I think you could move the content of those blocks back into this form theme file so that they are all in the same place